.github/workflows/docker-build-push.yaml

```
@@ -58,7 +86,8 @@ jobs:
       docker_secrets: mapbox_access_token=${{ secrets.NEXT_PUBLIC_MAPBOX_ACCESS_TOKEN }}
 
   build-kitty-krew:
-    needs: version
+    needs: [changes, version]
+    if: ${{ needs.changes.outputs.kitty_krew == 'true' }}
```

actionlint (1.7.4)

property "kitty_krew" is not defined in object type {findbobastore: string; kitty-krew: string; proompteng: string}

(expression)